crates/next_version/src/version_increment.rs

```diff
@@ -12,22 +12,8 @@ pub enum VersionIncrement {
     Prerelease,
 }
 
-fn is_there_a_custom_match(regex_option: Option<&Regex>, commits: &[ConventionalCommit]) -> bool {
-    if let Some(regex) = regex_option {
-        commits
-            .iter()
-            .any(|commit| custom_commit_matches_regex(regex, commit))
-    } else {
-        false
-    }
-}
-
-fn custom_commit_matches_regex(regex: &Regex, commit: &ConventionalCommit) -> bool {
-    if let CommitType::Custom(custom_type) = &commit.commit_type {
```

Comment:
we removed this check, so this is a breaking change because you can now override the default behavior of conventional commits types.
The new behavior is documented in the doc tests of this PR

Reply:
actually it shouldn't be a breaking change I think.